src/AzureRenderManager/WebApp/Views/Environments/View/Manager.cshtml

```diff
@@ -158,54 +158,54 @@
-                                        <label>Run Client as Service</label>
-                                    </div>
+                                    <label asp-for="DeadlineEnvironment.ServiceUser">Service Username</label>
+                                    <input asp-for="DeadlineEnvironment.ServiceUser" value="@Model.DeadlineEnvironment.ServiceUser" />
+                                    <span asp-validation-for="DeadlineEnvironment.ServiceUser" class=""></span>
+                                    <small>The existing user to run the Deadline client service under.  Typically this is required when the nodes will join a domain and you want to run the client service as a domain user.</small>
                                 </div>
-                                <div id="DeadlineServiceForm">
-                                    <div class="form-element">
-                                        <label asp-for="DeadlineEnvironment.ServiceUser">Service Username</label>
-                                        <input asp-for="DeadlineEnvironment.ServiceUser" value="@Model.DeadlineEnvironment.ServiceUser" />
-                                        <span asp-validation-for="DeadlineEnvironment.ServiceUser" class=""></span>
-                                        <small>The existing user to run the Deadline client service under.  Typically this is required when the nodes will join a domain and you want to run the client service as a domain user.</small>
-                                    </div>
-                                    <div class="form-element">
-                                        <label asp-for="DeadlineEnvironment.ServicePassword">Service Password</label>
-                                        <input asp-for="DeadlineEnvironment.ServicePassword" value="@Model.DeadlineEnvironment.ServicePassword" type="password" />
-                                        <span asp-validation-for="DeadlineEnvironment.ServicePassword" class=""></span>
-                                    </div>
+                                <div class="form-element">
+                                    <label asp-for="DeadlineEnvironment.ServicePassword">Service Password</label>
+                                    <input asp-for="DeadlineEnvironment.ServicePassword" value="@Model.DeadlineEnvironment.ServicePassword" type="password" />
+                                    <span asp-validation-for="DeadlineEnvironment.ServicePassword" class=""></span>
                                 </div>
+                            </div>
+                            <div class="form-element master-component">
+                                <label></label>
+                                <div class="form-check">
+                                    <input asp-for="DeadlineEnvironment.UseDeadlineDatabaseCertificate" type="checkbox">
+                                    <label>Deadline DB Requires a Certificate</label>
```

Comment:
Small 'r' for 'requires'. #superMinorNitpickNotReallyWorthMentioningTBH

Reply:
Updated to actually test that the hostname is resolvable.